### DISCUSSION TITLE
Add possibility to turn off Attributes checkbox in developer-tools/state by default

### DISCUSSION THREAD

**Discussion by gerard33** [home-assistant/frontend#7537] [Other feature requests] [answered]
I have a lot of entities, so every time I open developer-tools/state it takes around 5-6 seconds before the total list of entities is shown. After a search command it also takes a few seconds for the page to load when I remove the search filter.

I see that the loading is very fast when I uncheck the Attribute checkbox. The problem however is that this setting is not stored and disappears after e.g. a refresh of the page.

Feature request: store the setting of the Attribute checkbox in the Developer Tools - States page.

**Comment by KTibow** [comment]
This wouldn't be too hard with localStorage. I don't have this problem personally, but this sounds handy. Can anyone else verify that this is a good idea?

**Reply by BoneheadFraggle** [reply]
That would be just great. Then you can check the box the few times you actually want it instead.

**Reply by KTibow** [reply]
Does this code look good to you?
![image](https://user-images.githubusercontent.com/10727862/100530453-2e3d4980-31a7-11eb-8953-9ae63c2e844e.png)

**Reply by gerard33** [reply]
> That would be just great. Then you can check the box the few times you actually want it instead.

Exactly, without the attributes the page loads normal when having a lot of entities. After searching one can tick the box to show the attributes if necessary. Would be nice if this gets implemented.

@KTibow don't have any knowledge on that part of the code, so for me it looks good 😉. Thanks for actually making a code proposal 👍

**Comment by KTibow** [comment, marked as answer]
🚀 #8010 
Still need to test it on my dev instance.

**Comment by gerard33** [comment]
Thanks for the PR which is merged in https://github.com/home-assistant/frontend/pull/8448 to frontend 20210224.0. In HA 2021.3.0 the frontend version is 20210302.3.

I am running HA 2021.3.1 but the attribute checkbox is not saved for me in Vivaldi and Edge. After unchecking the checkbox and refreshing the page, the checkbox is checked again.

Am I missing something?

**Reply by KTibow** [reply]
Nope. Me too. After some messing around in devtools, it looks like it's not even setting it in localStorage.

**Reply by KTibow** [reply]
Make a new issue for this, and @mention me. That'll auto-subscribe me to it.

**Reply by gerard33** [reply]
Thanks! See https://github.com/home-assistant/frontend/issues/8572.

**Reply by spacegaier** [reply]
Had the same issue in Chrome and just created PR https://github.com/home-assistant/frontend/pull/8579 to fix it.

**Reply by gerard33** [reply]
Thanks!

**Reply by gerard33** [reply]
Works great now in HA 2021.3.3 👍
